test/e2e/specs/all-features-confirmation/record/delete-btn.spec.js

```diff
@@ -101,8 +101,9 @@ describe('View existing record,', function() {
                 chaisePage.waitForElement(element(by.id('tblRecord'))).then(function() {
                     return browser.executeScript('return chaiseConfig;');
                 }).then(function(chaiseConfig) {
-                    config = chaiseConfig
-                    return chaisePage.recordPage.getDeleteRecordButton().click()
+                    config = chaiseConfig;
+                    browser.sleep(1000);
```

Comment:
Maybe we want to wait for the delete button to be clickable using a `browser.wait()` then click the button.